packages/next-server/server/lib/path-match.ts

```diff
@@ -2,23 +2,22 @@
 // That's because, ^^^ package comes with very old version of path-to-regexp
 // So, it'll give us issues when the app has used a newer version of path-to-regexp
 // (When webpack resolving packages)
-var pathToRegexp = require('path-to-regexp')
+const pathToRegexp = require('path-to-regexp')
```

Comment:
Why there is no module import?

Reply:
Because i just converted the file without any unneeded changes.